### PR TITLE
[ci]: Update iroha2 stable/lts branch workflows triggers

### PR DESCRIPTION
**Pull request by BAStos525**
Update workflows triggers for iroha2 `stable/lts` branch

### CODE REVIEW

**Review by nxsaken — APPROVED** [review]

File: .github/workflows/iroha2-release.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: I2::Release::Publish
 
 on:
   push:
-    branches: [iroha2-stable, iroha2-lts]
+    branches: [stable, lts]
```
Comment (Erigara) [inline .github/workflows/iroha2-release.yml:5]: `iroha2-lts` wasn't renamed to `lts` as i can see

Comment (mversic) [reply .github/workflows/iroha2-release.yml:5]: tbh `lts` stuff quite is irrelevant now

Comment (BAStos525) [reply .github/workflows/iroha2-release.yml:5]: It's just a plug for a future possible `lts` branch that could be appeared. I can remove it.

Comment (BAStos525) [reply .github/workflows/iroha2-release.yml:5]: So, should it be pinned to the old `iroha2-lts` or to be removed completely?

Comment (mversic) [reply .github/workflows/iroha2-release.yml:5]: pin it to `iroha2-lts`, but we're going to remove it later

**Review by mversic — DISMISSED** [review]

File: .github/workflows/iroha2-release.yml (hunk 1 above)
Comment (BAStos525) [reply .github/workflows/iroha2-release.yml:5]: Okay, but step `name: Get tag from branch name` will now work for `iroha2-lts` branch

Comment (mversic) [reply .github/workflows/iroha2-release.yml:5]: it's not relevant, we're removing `lts` branch anyhow

Comment (BAStos525) [reply .github/workflows/iroha2-release.yml]: Done

**Review by nxsaken — APPROVED** [review]

**Review by mversic — APPROVED** [review]

**Pull request merged**